### PR TITLE
Modified node logic to return a 500 status code if data are returned …

### PR DESCRIPTION
**Pull request by devo132**
…from the java back end (such as an error message - indended output is none). Modified front end to check the response text from the node back end and react accordingly since error response handler was not running.

### CODE REVIEW

**Comment by steverydzmb** [comment]
Remove the debugging logs, but otherwise :shipit:

File: atlas-admin-frontend/js/controllers/feeds/console.js [hunk 1]
```diff
@@ -219,6 +219,12 @@ function($scope, $modalInstance, $q, Feeds, modalAction, $http, atlasHost) {
       
       Feeds.request('youview/bbc_nitro/action/revoke', 'post', payload).then(
       function (data, status) {
+        if (_.isObject(data)) {
```
Comment (cognitive-i) [inline atlas-admin-frontend/js/controllers/feeds/console.js]: So the server only returns a data object in the error path?  Shouldn't we be looking at the status?

File: atlas-admin-frontend/js/controllers/feeds/console.js [hunk 2]
```diff
@@ -219,6 +219,12 @@ function($scope, $modalInstance, $q, Feeds, modalAction, $http, atlasHost) {
...
+          $scope.resultMessage.body = 'The transaction could not be completed because of a server error';
+          $scope.resultMessage.class = 'error';
+          $scope.isBusy = false;
+          return;
```
Comment (cognitive-i) [inline atlas-admin-frontend/js/controllers/feeds/console.js]: Is the error logged anywhere?

File: atlas-admin-api/lib/gateways/feeds.js [hunk 3]
```diff
@@ -247,6 +247,11 @@ var feedsInterface = function() {
 
       action_res.on('end', function() {
         forceContentIntoQueue(uri).then(function () {
+          if (data.length > 0) {
```
Comment (cognitive-i) [inline atlas-admin-api/lib/gateways/feeds.js]: Why is this necessary over status check?
